lib/ui/home.dart

```diff
@@ -1,176 +1,156 @@
 import 'package:farmsmart_flutter/farmsmart_localizations.dart';
-import 'package:farmsmart_flutter/redux/app/app_state.dart';
-import 'package:farmsmart_flutter/ui/community/community_child.dart';
+import 'package:farmsmart_flutter/ui/bottombar/persistent_bottom_navigation_bar.dart';
+import 'package:farmsmart_flutter/ui/bottombar/tab_navigator.dart';
 import 'package:farmsmart_flutter/ui/discover/ArticleList.dart';
-import 'package:farmsmart_flutter/ui/home_viewmodel.dart';
-import 'package:farmsmart_flutter/ui/myplot/PlotList.dart';
+import 'package:farmsmart_flutter/ui/playground/data/playground_datasource_impl.dart';
 import 'package:farmsmart_flutter/ui/playground/playground_view.dart';
-import 'package:farmsmart_flutter/utils/assets.dart';
-import 'package:farmsmart_flutter/utils/colors.dart';
-import 'package:farmsmart_flutter/utils/dimens.dart';
-import 'package:firebase_dynamic_links/firebase_dynamic_links.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_redux/flutter_redux.dart';
-import 'profitloss/ProfitLossList.dart';
-import 'package:farmsmart_flutter/ui/playground/data/playground_datasource_impl.dart';
 
+import 'package:farmsmart_flutter/data/repositories/flamelink_repository_provider.dart';
+import 'myplot/PlotList.dart';
 
-/// Home "screen" route. Scaffold has all the app subcomponents available inside,
-/// like bottom bar or action bar.
-///
+class _Constants {
+  static final double bottomBarIconSize = 25;
 
-final cms = FlameLink(store: Firestore.instance, environment: Environment.development);
-final articleRepo = ArticlesRepositoryFlameLink(cms);
-final plotRepo = MockPlotRepository();
-
-class Home extends StatefulWidget {
-  @override
-  State<StatefulWidget> createState() {
-    return _HomeState();
-  }
+  static final myPlotSelectedIcon = 'assets/icons/my_plot_selected.png';
+  static final myPlotIcon = 'assets/icons/my_plot.png';
+  static final profitLossSelectedIcon = 'assets/icons/profit_loss_selected.png';
+  static final profitLossIcon = 'assets/icons/profit_loss.png';
+  static final discoverSelectedIcon = 'assets/icons/discover_selected.png';
+  static final discoverIcon = 'assets/icons/discover.png';
+  static final communitySelectedIcon = 'assets/icons/community_selected.png';
+  static final communityIcon = 'assets/icons/community.png';
 }
 
-class _HomeState extends State<Home> with WidgetsBindingObserver {
-  HomeViewmodel homeViewModel;
+class Home extends StatelessWidget {
+  FarmsmartLocalizations localizations;
+  final RepositoryProvider repositoryProvider;
+
+  Home({
+    Key key,
+    this.repositoryProvider,
+  }) : super(key: key);
 
   @override
-  void initState() {
-    super.initState();
-    WidgetsBinding.instance.addObserver(this);
+  Widget build(BuildContext context) {
+    localizations = FarmsmartLocalizations.of(context);
+
+    return PersistentBottomNavigationBar(
+      backgroundColor: Colors.white,
```

Comment:
maybe should be lifted to _Constants. or its own _Colors.navBarBackground at the top...

Reply:
Done